mlir/lib/Transform/AIRMiscPasses.cpp

```diff
@@ -894,8 +894,7 @@ Value tileChannelOpByFactor(air::ChannelInterface originalChanOp, int factor,
   Value zeroIdx = builder.create<arith::ConstantIndexOp>(loc, 0);
   if (!originalChanOp.getOffsets().empty()) {
     auto offsetDefOp = originalChanOp.getOffsets()[dim].getDefiningOp();
-    if ((offsetDefOp && isa<affine::AffineApplyOp>(offsetDefOp)) ||
-        isa<air::ExecuteOp>(offsetDefOp))
+    if (offsetDefOp && isa<affine::AffineApplyOp, air::ExecuteOp>(offsetDefOp))
```

Comment:
Could probably use `llvm::isa_and_present` here to avoid the `&&`

Reply:
Good point. I created this PR to apply the `isa_and_present` and `dyn_cast_if_present` in the codebase. https://github.com/Xilinx/mlir-air/pull/821 Thanks for the comment!